assets/src/edit-story/components/panels/design/alignment/alignment.js

```diff
@@ -219,97 +193,99 @@ function ElementAlignmentPanel({ selectedElements, pushUpdate }) {
   );
 
   return (
-    <Panel
+    <SpecialPanel
       name="alignment"
       canCollapse={false}
       ariaLabel={__('Alignment', 'web-stories')}
     >
       <ElementRow ref={ref}>
-        <WithTooltip title={__('Distribute horizontally', 'web-stories')}>
-          <Icon
+        <Tooltip
+          title={__('Distribute horizontally', 'web-stories')}
+          placement={PLACEMENT.BOTTOM}
+        >
+          <AlignmentButton
             disabled={!isDistributionEnabled}
             onClick={() => handleHorizontalDistribution(boundRect, pushUpdate)}
             aria-label={__('Horizontal Distribution', 'web-stories')}
             id={alignmentButtonIds[0]}
             onFocus={() => setCurrentButton(alignmentButtonIds[0])}
           >
-            <DistributeHorizontal />
-          </Icon>
-        </WithTooltip>
-        <WithTooltip title={__('Distribute vertically', 'web-stories')}>
-          <Icon
+            <Icons.DistributeHorizontal />
+          </AlignmentButton>
+        </Tooltip>
+        <Tooltip title={__('Distribute vertically', 'web-stories')}>
+          <AlignmentButton
             disabled={!isDistributionEnabled}
             onClick={() => handleVerticalDistribution(boundRect, pushUpdate)}
             aria-label={__('Vertical Distribution', 'web-stories')}
             id={alignmentButtonIds[1]}
             onFocus={() => setCurrentButton(alignmentButtonIds[1])}
           >
-            <DistributeVertical />
-          </Icon>
-        </WithTooltip>
-        <SeparateBorder />
-        <WithTooltip title={__('Align left', 'web-stories')} shortcut="mod+{">
-          <Icon
+            <Icons.DistributeVertical />
+          </AlignmentButton>
+        </Tooltip>
+        <Tooltip title={__('Align left', 'web-stories')} shortcut="mod+{">
+          <AlignmentButton
             onClick={() => handleAlign('left', boundRect, pushUpdate)}
             aria-label={__('Justify Left', 'web-stories')}
             id={alignmentButtonIds[2]}
             onFocus={() => setCurrentButton(alignmentButtonIds[2])}
           >
-            <AlignLeft />
-          </Icon>
-        </WithTooltip>
-        <WithTooltip title={__('Align center', 'web-stories')} shortcut="mod+H">
-          <Icon
+            <Icons.AlignLeft />
+          </AlignmentButton>
+        </Tooltip>
+        <Tooltip title={__('Align center', 'web-stories')} shortcut="mod+H">
```

Comment:
These shortcuts don't seem to work for me btw -- does it work for you?

Reply:
They never worked for me. Can confirm the same non-function in main for me at least, so I'll file it as a new bug (if it doesn't already exist).

Reply:
Filed in #6595.